docs/intro.rst

```diff
@@ -181,16 +190,6 @@ Here is some examples to make things clear::
 
     >>> t.String().check('werwerwer')
     'werwerwer'
-    >>> t.String(regex='^\s+$').check('   ')
```

Comment:
It might be good to leave `t.String(...) & t.Regexp(..)` example here. As it's quite common for working with strings.